client/blocks/signup-form/index.jsx

```diff
@@ -152,6 +163,15 @@ class SignupForm extends Component {
 		recordTracksEvent( 'calypso_signup_back_link_click' );
 	};
 
+	componentDidUpdate() {
```

Comment:
Could we achieve the same effect by the following lines?
```
 componentDidMount() {
        this.emailRef.current.focus();
  }
```
It looks like all that we want to achieve here is to auto focus on the email field without using the `autoFocus` attribute which will trigger a11y warning, so I thought we might be able to achieve the same by using `componentDidMount()`.

Reply:
Oh my that is what I meant to put 🤭 , Thanks for picking this up.

Reply:
Or to just add `autoFocus` and disable the a11y warning if we're just going to do what the warning is warning us about anyway.